woocommerce/woocommerce-android
# [Order Creation] add shipping

Closes: #5697 


### Description
This adds support for adding shipping fees to the created order.
The edit flow and payment section update will be handled in a future PR.

### Testing instructions
1. Make sure the feature flag `ORDER_CREATION_M2` is enabled.
2. Open order creation form.
3. Add a product.
4. Notice the button "Add shipping" on the payment section.
5. Click on "Add shipping".
6. Enter an amount and a name.
7. Notice that the draft order has been updated, the taxes value may be updated if the taxes settings are applied to shipping too.
8. Finish the order creation.
9. Confirm the created order has the shipping fees.

### Images/gif
<img width="398" alt="Screen Shot 2022-02-11 at 15 57 18" src="https://user-images.githubusercontent.com/1657201/153614834-9f300f54-0076-4604-9d17-4a6bdc4f7f54.png">
